Documentation/design-docs/event-counter.md

```diff
@@ -63,13 +66,14 @@ On the wire EventCounter and PollingCounter both produce an event with name "Eve
         DisplayName: "Request Bytes"
         Name: "request-bytes",
         Mean: 12.32,
-        StandardDeviation: 2.45
-        Count: 7
-        Min: -3.4
-        Max: 22.98
-        IntervalSec: 1.00324
-        Series: "Interval=1"
-        CounterType: "Mean"
+        StandardDeviation: 2.45,
+        Count: 7,
+        Min: -3.4,
+        Max: 22.98,
+        IntervalSec: 1.00324,
+        Series: "Interval=1",
+        CounterType: "Mean",
+        MetaData: "key1=value1,key1=value1,key1=value1"
```

Comment:
nit: `"key1=value1,key1=value1,key1=value1"` Is duplicating the `key1=value1` intentional?
*same below.

Reply:
no... that was copy-pasted. Will address it now. Thx!